src/internal/components/drag-handle-wrapper/portal-overlay.tsx

```diff
@@ -36,19 +36,23 @@ export default function PortalOverlay({
       // It could be that the portal hasn't been attached to the DOM yet - ensure the ref exists and is attached DOM tree.
       if (ref.current && document.body.contains(ref.current)) {
         const isRtl = getIsRtl(ref.current);
-        const { insetInlineStart, insetBlockStart, inlineSize, blockSize } = getLogicalBoundingClientRect(track);
+        const { insetInlineStart, insetBlockStart, inlineSize, blockSize } = getLogicalBoundingClientRect(
+          track.current!
```

Comment:
`!` becaues we know that `track.current` won't be null because we checked it in line 26.
Not sure why TypeScript doesn't consider that check above.